lib/typings.d.ts

```diff
@@ -31,6 +32,7 @@ export type RecordTelemetryFn = (
 export type FraudAssesment =
   | 'OK'
   | 'INVALID_TASK'
+  | 'TASK_NOT_ALLOWED'
```

Comment:
Great question! In the current code:

- INVALID_TASK means the task does not match any of the 1000 tasks defined for the round.
- TASK_NOT_ALLOWED means the task is valid for the round, but this node is not allowed to perform it.

I'd like to keep the distinction. INVALID_TASK is typically used for measurements that are committed to the wrong round. Typically, all measurements in the first batch published after a new round starts are measurements for tasks from the previous round because spark-api & checker nodes haven't noticed the new round yet. 

I agree to find more descriptive names 👍🏻

Reply:
Got it! What about

- `TASK_NOT_FOUND`
- `TASK_WRONG_NODE`

Reply:
What do you think about this?

- `TASK_NOT_IN_ROUND`
- `TASK_NOT_FOR_NODE`

On the second thought, `TASK_WRONG_NODE` works too 👍🏻